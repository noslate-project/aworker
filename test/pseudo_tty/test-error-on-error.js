'use strict';

addEventListener('error', () => {
  throw 'fatal';
});

throw new Error('foobar');
