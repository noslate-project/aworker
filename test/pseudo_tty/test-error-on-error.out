Uncaught Exception on "error": fatal
